DER/DerivativesContracts/Options.rdf

```diff
@@ -95,22 +130,25 @@
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Quantities/QuantitiesAndUnits/"/>
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Relations/Relations/"/>
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/Analytics/"/>
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/IND/ForeignExchange/ForeignExchange/"/>
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/IND/Indicators/Indicators/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/IND/MarketIndices/BasketIndices/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/Bonds/"/>
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/ExerciseConventions/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/Equities/EquityInstruments/"/>
+		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/Securities/SecuritiesListings/"/>
 		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/Options/"/>
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Provisional"/>
 	</owl:Ontology>
 	
-	<owl:Class rdf:about="&fibo-der-der-opt;AbsolutePriceStrikeSpecification">
-		<rdfs:subClassOf rdf:resource="&fibo-der-der-opt;StrikePrice"/>
-		<rdfs:label xml:lang="en">absolute price strike specification</rdfs:label>
-		<owl:disjointWith rdf:resource="&fibo-der-der-opt;PercentageStrikePriceSpecification"/>
-		<skos:definition xml:lang="en">Expression of the strike price as an absolute monetary price at which the option is to be struck.</skos:definition>
+	<owl:Class rdf:about="&fibo-der-drc-bsc;DerivativesClearingOrganization">
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">For options, the derivatives clearing organization is responsible for clearing transactions for put and call options on common stocks and other equity issues, stock indexes, foreign currencies, interest rate composites and single-stock futures. e.g. Options Clearing Corporation (US). In Canada the equivalent is In the US, the Options Clearing Corporation defines defaults e.g. lot sizes. This applies to equity options, index options, flex options, yield based options and short dated options. It does not apply to other types of option. Its roles are: 1. Clearing 2. Advocate for sound business practice. Similar to regulator though not officially one. Has a role in risk management Full list from OCC website: OCC clears transactions for put and call options on common stocks and other equity issues, stock indexes, foreign currencies, interest rate composites and single-stock futures</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
OCC should be modeled as an individual in its own right with this level of info and more (e.g. it's under CFTC).